recipes/glsl-optimizer/all/conanfile.py

```diff
@@ -0,0 +1,38 @@
+import os
+import glob
+from conans import ConanFile, CMake, tools
+
+class GLSLOptimizerConan(ConanFile):
+    name = "glsl-optimizer"
+    description = "GLSL optimizer based on Mesa's GLSL compiler. Used in Unity for mobile shader optimization."
+    homepage = "https://github.com/aras-p/glsl-optimizer"
+    url = "https://github.com/conan-io/conan-center-index"
+    topics = ("conan", "glsl", "opengl")
+    license = "MIT"
+    settings = "os", "arch", "compiler", "build_type"
+    generators = "cmake"
+    exports_sources = "CMakeLists.txt"
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def requirements(self):
+        self.requires("opengl/system")
```

Comment:
Can you have a look and check whether opengl is really needed?
I think opengl is only needed for the tests. The main optimizier should not need opengl.
It should be possible to disable building the tests.